benchmarks/svm.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from sklearn import svm
+
+from .common import Benchmark
+from .datasets import _synth_classification_dataset
+
+
+def getOptimalCacheSize(numFeatures):
+    byte_size = np.empty(0, dtype=np.double).itemsize
+    optimal_cache_size_bytes = numFeatures * numFeatures * byte_size
+    eight_gb = byte_size * 1024 * 1024 * 1024
```

Comment:
nitpick: `8 GB` is actually `8e9` bytes. What you `byte_size * 1024 * 1024 * 1024` is actually 8 GiB.